### DISCUSSION TITLE
EWW_TIME language

### DISCUSSION THREAD

**Discussion by gh0stzk** [elkowar/eww#943] [Q&A]
Hello, a fast question, i have this label

`(label :text "${formattime(EWW_TIME, "%A, %d, %B")}")`
That shows me Monday, 2, October

But im in mexico, spanish language, if i add `(label :text "${formattime(EWW_TIME, "%A, %d, %B" "America/Mexico_City")}")`
The result is the same in english, how i can made it shows in spanish?

I use arch linux, with es_MX locales only.

**Comment by itsanonmelody** [comment]
I checked the Rust documentation for chrono. Localisation support for date and time is not stable and disabled by default. Unfortunately, it seems like eww doesn't enable this feature, so it is not possible to have the days and months in a different language using the `formattime` function. You would need to achieve this through the `date` shell command.

**Comment by itsanonmelody** [comment]
Here's the PR for this exact issue #901

**Reply by gh0stzk** [reply]
Thankyou for your answer, i will do with the date command then. Thanks
